internal/service/opensearchserverless/vpc_endpoint_test.go

```diff
@@ -47,7 +47,7 @@ func TestAccOpenSearchServerlessVPCEndpoint_basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckVPCEndpointExists(ctx, resourceName, &vpcendpoint),
 					resource.TestCheckResourceAttr(resourceName, "subnet_ids.#", acctest.Ct1),
-					resource.TestCheckResourceAttr(resourceName, "security_group_ids.#", acctest.Ct1),
+					//resource.TestCheckResourceAttr(resourceName, "security_group_ids.#", acctest.Ct1),
```

Comment:
Looks like these checks may have been commented out during testing. Can we uncomment it?